resources/views/photos/album.blade.php

```diff
@@ -19,20 +19,22 @@
             <a href="{{route("photo::albums")}}" class="btn btn-success float-start me-3">
                 <i class="fas fa-list"></i> <span class="d-none d-sm-inline">Album overview</span>
             </a>
+            @if(!isset($liked))
             @can('protography')
                 <a href="{{route("photo::admin::edit", ['id' => $album->id])}}"
                    class="btn btn-success float-start me-3">
                     <i class="fas fa-edit"></i> <span class="d-none d-sm-inline">Edit album</span>
                 </a>
             @endcan
+            @endif
             <div class="p-1 m-1 fw-bold">
                 {{ $album->name }} ({{ date('M j, Y', $album->date_taken) }})
             </div>
         </div>
 
 
         <div class="card-body">
-            @if($album->mayViewAlbum(Auth::user()))
+            @if(isset($liked)||$album->mayViewAlbum(Auth::user()))
```

Comment:
I do not think users should be able to view liked pictures from albums they are not allowed to view.